source/team.html.erb

```diff
@@ -9,86 +9,96 @@ responsive: true
   Ember is an Open Source project that relies on the tireless support of individual contributors. These are the teams 
   that guide the development and instruction of Ember.js. 
 </p>
+<section class="section-team">
+ <h2 class="text-center">The Steering Committee</h2>
 
-<section class="team section">
-  <h2 class="text-center">The Core Team</h2>
+<p class="text-center">
+    The Steering Committee is responsible for the overall governance of the Ember project. 
+</p>
 
-  <% core = data.team.select { |m| m.teams.include?('core') } %>
+<% steering = data.team.select { |m| m.teams.include?('steering') } %>
+  <ul class="list-team">
+    <% steering.each do |user| %>
+      <li class="list-team-item">
+        <img src="/images/team/<%= user.image %>" alt="<%= user.name %>" class="team-member-image">
+        <div class="team-member-name">
+          <%= user.name %>
+        </div>
+        <ul class="team-member-social">
+          <li>
+            <a class="team-member-twitter" href="<%= user.twitter %>" aria-label="<%= user.name %> Twitter">
+              <i class="icon-twitter" aria-hidden="true"></i>
+            </a>
+          </li>
+          <li>
+            <a class="team-member-github" href="<%= user.github %>" aria-label="<%= user.name %> Github Profile">
+              <i class="icon-github" aria-hidden="true"></i>
+            </a>
+          </li>
+        </ul>
+      </li>
+    <% end %>
+  </ul>
+</section>
 
-  <ul class="headshots">
-    <% core.each do |user| %>
-      <li>
-        <a href="<%= user.github %>" rel="nofollow" class="avatar">
-          <img src="/images/team/<%= user.image %>" alt="<%= user.name %>">
-        </a>
+<hr/>
 
-        <p class="name">
-          <%= user.name %>
-        </p>
+<section class="section-team">
+  <h2 class="text-center">The Ember.js Core Team</h2>
```

Comment:
Do we want some info about what the core team does here, now that it's been modified a bit?